{"cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2024-48768", "ASSIGNER": "cve@mitre.org"}, "problemtype": {"problemtype_data": [{"description": []}]}, "references": {"reference_data": [{"url": "http://appinventoraigooglealmandocontrol.com", "name": "http://appinventoraigooglealmandocontrol.com", "refsource": "", "tags": []}, {"url": "http://www.almando.com/", "name": "http://www.almando.com/", "refsource": "", "tags": []}, {"url": "https://github.com/HankJames/Vul-Reports/blob/main/FirmwareLeakage/appinventor.ai_google.almando_control/appinventor.ai_google.almando_control.md", "name": "https://github.com/HankJames/Vul-Reports/blob/main/FirmwareLeakage/appinventor.ai_google.almando_control/appinventor.ai_google.almando_control.md", "refsource": "", "tags": []}, {"url": "https://www.almando.com/media/firmware/almando.json", "name": "https://www.almando.com/media/firmware/almando.json", "refsource": "", "tags": []}]}, "description": {"description_data": [{"lang": "en", "value": "An issue in almaodo GmbH appinventor.ai_google.almando_control 2.3.1 allows a remote attacker to obtain sensitive information via the firmware update process"}]}}, "configurations": {"CVE_data_version": "4.0", "nodes": []}, "impact": {}, "publishedDate": "2024-10-11T20:15Z", "lastModifiedDate": "2024-10-15T21:35Z"}